### ISSUE TITLE
Codeformer settings not available?

### ISSUE THREAD

**Issue by M4CC** [chaiNNer-org/chaiNNer#1475]
Hello,

Is there a way to tweak Codeformer settings within Chainner? 

I'm getting much better results with this implementation: https://huggingface.co/spaces/sczhou/CodeFormer probably because I can tweak the fidelity settings there but I can't seem to do it within Chainner.

Is there some way to select those within the interface?

Thank you!

**Comment by M4CC** [comment]
Thank you very much for your reply.

Indeed I'm passing the upscaled background from realesrgan-x4plus to the upscale face's "upscaled background" and it works great. For the upscale face's "image" though I realized I need to pass the base image otherwise it doesn't get retouched (I guess that's how it is intended), then I just upscale everything together a bit more within the upscale face's module.

I realized that using GFPGANv1.3 as the upscale face's model works much better as it is but I don't like the output faces as much as codeformer's, they're a bit less expressive but it does retouch them quite perfectly. Codeformer really lacks that fidelity setting I think to make them perfect, I am using "0" all the time because since it's AI I don't need any fidelity.

I am wondering if there's some possibility to "bake the codeformer model before feeding it to chaiNNer to keep that level of fidelity while there's no option to tweak it within it?
